lib/quby/questionnaires/entities/charting/chart.rb

```diff
@@ -10,9 +10,20 @@ class Chart
           attribute :key,                        Symbol
           attribute :title,                      String
           attribute :plottables,                 Array
-          # if categories are defined, plottable values should correspond to values from this array
-          # and the graph will  be plotted with matching y axis categories
+          # if y_categories are defined, plottable values should correspond to values from this array
+          # and the graph will be plotted with corresponding y axis categories
+          # example (icl_r): ["Zeer laag", "Laag", "Gemiddeld", "Hoog", "Zeer Hoog"] (caution, capitalization oddity)
+          # NB: only implemented for bar charts on the roqua side
           attribute :y_categories,               Array
+          # if y_range_categories are defined, plottable values should fall in the ranges that compose the keys of this
+          # hash. The chart will label these ranges of y_values with the corresponding value in the hash on the y axis.
+          # example:
+          # {(0..30) => "Zeer laag", (30..40) => "Laag", (40..60) => "Gemiddeld",
+          # (60..70) => "Hoog", (70..100) => "Zeer hoog"}
```

Comment:
Deze paar regels code kunnen natuurlijk makkelijk aangepast, probleem is dat op dat moment de questionnaires al geschreven zijn en in roqua de code om hier gebruik van te maken en omdat het een simpele hash is kun je niet zomaar iets toevoegen. Ja we hebben as_json en value_to_category nu nog niet nodig, dus hoeven we ook nog niet te schrijven, maar als we dat later wel toe willen voegen moet er opeens in meerdere repo's code aangepast worden. 
Daarbij zeg je hier dat je including ranges gebruikt om roqua code makkelijker te maken, terwijl het semantisch volledig incorrect is (en niet nodig omdat .last ook bestaat). 

niet aansluitende ranges zijn niet nuttig zeg je (zei ik ook al), maar er is geen validatie of de ranges aansluitend zijn. Geen validaties of de ranges including zijn geschreven (of excluding als het aan mij ligt). 

Ik noem het houtje touwtje omdat het net genoeg is om voor de huidige usecase te werken, maar niet perse mooi is, noch robust, noch makkelijk uitbreidbaar (met behoud van oude interfaces).

Maar prima, als verder niemand inspringt laten we het zo.

doch wel graag exclusive ranges.